rmf_traffic_ros2/src/update_participant/main.cpp

```diff
@@ -21,7 +21,7 @@
 #include <rclcpp/node.hpp>
 #include <rclcpp/executors.hpp>
 
-#include "../../rmf_traffic_ros2/src/rmf_traffic_ros2/schedule/internal_YamlSerialization.hpp"
+#include "../src/rmf_traffic_ros2/schedule/internal_YamlSerialization.hpp"
```

Comment:
I'm not quite sure why different environments are inconsistent about finding the header, but both of these paths look unreliable to me.

I believe we want this, to get the path of the header, relative to this `main.cpp` file:

```
#include "../rmf_traffic_ros2/schedule/internal_YamlSerialization.hpp"
```

Reply:
True, my path was wrong. For some reason the path seemed to be working.

Fixed in https://github.com/open-rmf/rmf_ros2/pull/67/commits/f8375ab9e7ffa3ffca5e0f591ed3beb30d901823